src/api/app/controllers/webui/projects/project_configuration_controller.rb

```diff
@@ -5,12 +5,14 @@ class ProjectConfigurationController < WebuiController
       after_action :verify_authorized, only: :update
 
       def show
-        sliced_params = params.slice(:rev).permit!
-        @content = @project.config.content(sliced_params.to_h)
+        result = ::ProjectConfigurationService::ProjectConfigurationPresenter.new(@project, params).call
         switch_to_webui2
-        return if @content
-        flash[:error] = @project.config.errors.full_messages.to_sentence
-        redirect_to projects_path(nextstatus: 404)
+        if result.valid?
+          @content = result.config
+        else
+          flash[:error] = @project.config.errors.full_messages.to_sentence
```

Comment:
`@project.config.errors.full_messages.to_sentence` is exactly what `result.errors` returns